app/templates/main/manageServiceLearningFaculty.html

```diff
@@ -119,6 +119,9 @@ <h5 class="ms-3">There are no approved courses for {{ term.description }}.</h5>
           <div class="float-end">
             <a  class="btn btn-primary mr {{downloadAndSend}}" href="mailto:{{(';'.join(instructorEmails))}}?subject=Designated Service-Learning Courses: List of recommended courses for {{term.description}}&body=Hello,%0D%0A%0D%0AI’m writing to submit the list of courses recommended for the Service-Learning Course Designation (meeting the ALE requirement) for the upcoming semester; this list is ready to be shared with COGE. Please find the list of courses in the attached document. The proposals have been reviewed and approved by the Service-Learning Faculty Fellow and Director of CELTS (me).%0D%0A%0D%0AIn addition to these courses, all the courses that have received the Permanent Service-Learning Course Designation that will be taught in the upcoming semester should also have the Service-Learning Course Designation.%0D%0A%0D%0AThank you,%0D%0A%0D%0AAshley" target="_blank"> Send Recommendation </a>
           </div>
+            <div class="float-end">
+              <a  class="btn btn-primary mr {{downloadAndSend}}" href="mailto:registrar@berea.edu?subject=Requesting%20Student%20Service-Learning%20Course%20participation&body=ADD TEXT" target="_blank"> Email Registar </a>
```

Comment:
If we are making these changes, we should actually finish them, which would mean creating the body of the email.

Reply:
I suggested the placeholder since I thought it would be good to ask Ashley what she would like put in there and then adding it in.